spec/javascripts/LoginPage/Login3scaleForm.spec.jsx

```diff
@@ -28,17 +28,31 @@ it('should render HiddenInputs component', () => {
 })
 
 it('should set username state', () => {
+  const event = {
+    currentTarget: {
+      name: 'username',
+      value: 'Bob',
+      type: 'text'
+    }
+  }
   const wrapper = mount(<Login3scaleForm {...props}/>)
-  jest.spyOn(wrapper.instance(), 'handleTextInputUsername')
-  wrapper.instance().handleTextInputUsername('foo')
-  expect(wrapper.instance().handleTextInputUsername).toHaveBeenCalled()
-  expect(wrapper.state().username).toEqual('foo')
+  jest.spyOn(wrapper.instance(), 'handleInputChange')
+  wrapper.instance().handleInputChange('Bob', event)
+  expect(wrapper.instance().handleInputChange).toHaveBeenCalled()
```

Comment:
It's been called literally in the previous line. Instead of that, find the input and simulate a change there, then asset the spy.